gix/src/open/repository.rs

```diff
@@ -58,6 +59,8 @@ impl ThreadSafeRepository {
     ///
     /// Note that opening a repository for implementing custom hooks is also handle specifically in
     /// [`open_with_environment_overrides()`][Self::open_with_environment_overrides()].
+    #[momo]
+    #[allow(unused_mut)]
```

Comment:
`#[allow(unused_mut)]` Is probably a side-effect of `momo` because it just passes `options` on without ever needing it to be `mut`. Since it would see the `mut` I suppose it should be the one automatically adding the lint configuration.